pkg/operator/apis/aro.openshift.io/v1alpha1/cluster_types.go

```diff
@@ -48,6 +48,30 @@ type InternetCheckerSpec struct {
 	URLs []string `json:"urls,omitempty"`
 }
 
+type OperatorFlags map[string]string
+
+func (f OperatorFlags) GetBoolean(key string) (val bool, exists bool) {
+	v, ext := f[key]
+	if ext {
+		// Only accept a literal true, rather than accepting anything other than
+		// literal false as true
+		// TODO: Is this the best behaviour?
```

Comment:
One thing that kind of worries me is value abuse ("false - we decided to disable it because…"), but it's a matter of SRE team discipline for now.
I don't know if we should have a validator to check for "true"/"false", since "True" will be false.

Reply:
ideally we would have a system that sets this for us and not manually :) -- one day